mappings/net/minecraft/client/gui/screen/ingame/MerchantScreen.mapping

```diff
@@ -7,6 +7,21 @@ CLASS net/minecraft/class_492 net/minecraft/client/gui/screen/ingame/MerchantScr
 	FIELD field_26570 SEPARATOR_TEXT Lnet/minecraft/class_2561;
 	FIELD field_26571 DEPRECATED_TEXT Lnet/minecraft/class_2561;
 	FIELD field_2950 TEXTURE Lnet/minecraft/class_2960;
+	FIELD field_32354 TEXTURE_WIDTH I
+	FIELD field_32355 TEXTURE_HEIGHT I
+	FIELD field_32357 XP_BAR_X_OFFSET I
+	FIELD field_32358 TRADE_LIST_AREA_Y_OFFSET I
+	FIELD field_32359 FIRST_BUY_ITEM_X_OFFSET I
+	FIELD field_32360 SECOND_BUY_ITEM_X_OFFSET I
+	FIELD field_32361 SOLD_ITEM_X_OFFSET I
```

Comment:
```suggestion
	FIELD field_32357 XP_BAR_OFFSET_X I
	FIELD field_32358 TRADE_LIST_AREA_OFFSET_Y I
	FIELD field_32359 FIRST_BUY_ITEM_OFFSET_X I
	FIELD field_32360 SECOND_BUY_ITEM_OFFSET_X I
	FIELD field_32361 SOLD_ITEM_OFFSET_X I
```

Reply:
Those offsets don't have a corresponding x/y component, so I think they are more like "x/y offset" than "x/y of the offset"